proposals/declaration-expressions.md

```diff
@@ -0,0 +1,78 @@
+# Declaration expressions
```

Comment:
This section at the top is not needed in the proposal. It is used in the championed issue referring to the proposal.

Reply:
Removed.